src/main/webapp/src/components/CourseInputForm.vue

```diff
@@ -409,10 +401,15 @@ export default {
         parseValidDate: function(val) {
             const date = parseDate(val);
             return isValid(date) ? date.toISOString() : null;
-        },
-        onCategorySelect(updatedCategories) {
-            this.selectedCategories = updatedCategories;
         }
+    },
+    mounted: function() {
+        getAllCategories()
```

Comment:
@cbuettgentarent  getCategories instead of getAllCategories would do. We have getCourses